src/test/java/org/apache/commons/crypto/cipher/AbstractCipherTest.java

```diff
@@ -147,6 +153,75 @@ public void cryptoTest() throws Exception {
         }
     }
 
+    @Test
+    public void testNullTransform() throws Exception {
+        for (String transform : transformations) {
```

Comment:
`transform` is not used in the body, is the loop really needed?

(I guess not. Then you can also use `@Test(expected=RuntimeException.class)`.)